src/features/form/validations.jsx

```diff
@@ -35,6 +35,19 @@ const isRequired = value => value ? undefined : <LocalizedMessage
   id="reactcomponents.value.required"
   defaultMessage="Required" />;
 
+const allowDecimals = value => value => {
```

Comment:
this should really be called disallowDecimals or preventDecimals, or something like that, right?